pyproject.toml

```diff
@@ -71,7 +71,10 @@ pandas = [
     {version = "<2.0", python = "<3.9"},
     {version = ">1.0", python = ">=3.9"}
 ]
-networkx = ">=2.8.5"
+networkx = [
```

Comment:
needs to bump the networkx dependency since the find_minimal_d_separator  algorithm wasnt available in release 2.8.5 (see https://github.com/networkx/networkx/pull/5898) - this requires python>=3.10, which means our generalized adjustment will only be available to users with python >=3.10

Reply:
That should be okay. This edit makes sense.